.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow handles releasing all of the packages in samples.
+
+name: publish
+
+on: workflow_dispatch  # Manually trigger.
+
+permissions:
+  contents: write  # For checkout and tag.
+  packages: write  # For publish.
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    # Environment specific to releasing so we can isolate the npm token.
+    environment: release  
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: NPM install
+      run: npm install
+
+    - name: Build
+      run: npm run build -- -- -- --skip-lint
+
+    - name: Test
+      run: npm run test -- -- -- --skip-lint
+
+    - name: Publish
+      env:
+        NODE_AUTH_TOKEN: ${{ secrets.RELEASE_BACKED_NPM_TOKEN }}
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: npm lerna publish --conventional-commits --create-release github --yes
```

Comment:
is this line correct? it should either be `lerna publish` or `npx lerna publish` I would think? (in other words should the `npm` be there in the front?

Reply:
Thanks for the catch, fixed!